packages/watchexec/build.sh

```diff
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/watchexec/watchexec
 TERMUX_PKG_DESCRIPTION="Executes commands in response to file modifications"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=1.18.6
+TERMUX_PKG_VERSION=1.18.8
 TERMUX_PKG_SRCURL=https://github.com/watchexec/watchexec/archive/refs/tags/cli-v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=d5cfd45ba05ac380e8ff7c3122d38b408a1cdb550c71036739f8c03a51117e84
+TERMUX_PKG_SHA256=1e450a8e13ccbddee86b9226798a513ea960c396a73d10964d224270f4843d4b
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_AUTO_UPDATE_TAG_REGEXP="\d+\.\d+\.\d+"
```

Comment:
this regex is probably not working (I don't know how to write regex) because [auto-update CI](https://github.com/termux/termux-packages/runs/5589872939?check_suite_focus=true#step:3:1536) returns 404 Not Found

![image](https://cdn.magicuser.cf/3qI12J7.png)

Reply:
Not working because it also traps patterns like `lib-v2.0.0-pre.13`. As a result, it tries to fetch `cli-v2.0.0`, which does not exist.

FYI, the same is happening for `libmsgpack` package.